src/main/java/io/orangebeard/client/entity/Attribute.java

```diff
@@ -14,13 +14,14 @@ public class Attribute {
     private String value;
 
     /**
-     * New attribute. Colons and Semicolons are removed because these are special values.
+     * New attribute.
      */
     public Attribute(String key, String value) {
-        this.key = key.replace(":", "").replace(";", "").trim();
-        this.value = value.replace(":", "").replace(";", "").trim();
+        this.key = key.trim();
+        this.value = value.trim();
     }
 
+
     /**
```

Comment:
Shouldn't the colons and semicolons be kept here, as well?

Reply:
I think removing colons and semicolons is not necessary in any use case. If we'd want to process them, we should explicitly do so. I'll update the value-only variant as well